### PR TITLE
Add debug log to output grpc request contents

### PR DESCRIPTION
**Pull request by utahta**
SSIA
https://github.com/mercari/grpc-federation/issues/160

### CODE REVIEW

File: generator/templates/eval.go.tmpl [hunk 1]
```diff
@@ -111,6 +111,9 @@ grpcfed.EvalDef({{ .Ctx }}, value, grpcfed.Def[{{ $def.Type }}, *localValueType]
 		}
 		{{- end }}
 		{{- end }}
+		{{- if $def.IsCall }}
+		s.logger.DebugContext(ctx, "call {{ $def.MethodFQDN }}", slog.Any("call_request", s.logvalue_{{ $def.LogValueRequestType }}(args)))
```
Comment (goccy) [inline generator/templates/eval.go.tmpl:115]: Here, I want to specify the FQDN of the request message

e.g.) `slog.Any("pkg.FooRequest", s.logvalue_Pkg_FooRequest(args))`

File: generator/code_generator.go [hunk 2]
```diff
@@ -2454,6 +2461,17 @@ func (d *VariableDefinition) RequestType() string {
 	return ""
 }
 
+func (d *VariableDefinition) LogValueRequestType() string {
+	expr := d.VariableDefinition.Expr
+	switch {
+	case expr.Call != nil:
+		return fullMessageName(expr.Call.Request.Type)
+	case expr.Message != nil:
```
Comment (goccy) [inline generator/code_generator.go:2469]: Since it will always be called under `$def.IsCall` in template, is this condition actually not evaluated ?

Comment (utahta) [reply generator/code_generator.go:2469]: Yes, this condition is not actually evaluated. I did the same as RequestType(), but we can remove it if not needed.

Comment (goccy) [reply generator/code_generator.go:2469]: Sure, Let's remove it then !

**Review by goccy — APPROVED** [review]
LGTM !!!

**Pull request merged**